rotkehlchen/db/settings.py

```diff
@@ -309,6 +322,24 @@ def read_boolean(value: str | bool) -> bool:
     )
 
 
+def _deserialize_evm_indexers_order(
+        value: str | dict[str, list[str]],
+) -> dict[ChainID, Sequence[EvmIndexer]]:
+    """Deserialize per-chain indexer order mapping."""
+    indexers = json.loads(value) if isinstance(value, str) else value
```

Comment:
Is it possible to have two different functions here to avoid this if check? We avoid this dynamic argument type overloading at runtime.


Or ... is this only used at schema validation in which case yes all checks should be exhaustive?

Same questions for the rest of the checks below